packages/core/src/components/showcase/StoryList.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import {
+  makeStyles,
+  useMediaQuery,
+  useTheme,
+  GridList,
+  GridListTile,
+} from "@material-ui/core";
+
+import SimpleBarReact from "simplebar-react";
+
+import StoryCard from "./StoryCard";
+
+import "simplebar/dist/simplebar.min.css";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexWrap: "wrap",
+    justifyContent: "space-around",
+    overflow: "hidden",
+    /**
+     * The mixture of rem and px set by GridList is
+     * causing height issues resulting in overflow.
+     * Stick to px for this component to function correctly.
+     */
+    height: `${theme.height}px`, // 23.125rem
```

Comment:
As above, don't use `theme.height`, `theme.width`, etc., since we're going to have a lot of components & we won't know which component does `theme.height` refer to. Use props.

Just in case: [Here](https://material-ui.com/styles/basics/#adapting-the-higher-order-component-api) you can see how to pass props to styles as adapt style based on current prop values.